docs/understanding-config-files.md

```diff
@@ -0,0 +1,27 @@
+
+# Running spark-bench from a Config File
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+
+You can do everything available in the CLI and more by running spark-bench through a configuration file.
+
+A configuration file defines a Spark context with one or many suites, each with one or many workloads. Suites
```

Comment:
I was thinking along the same lines.  In my documentation for Historian, I go through an illustrative sample config file line by line and try to explain the decisions involved in each.  I wonder if something like that may be appropriate here or in "Understanding spark-bench"....